CondTools/SiPixel/test/SiPixelTemplateDBObjectUploader_cfg.py

```diff
@@ -62,7 +62,6 @@
 options.parseArguments()
 
 MagFieldValue = 10.*options.MagField #code needs it in deciTesla
-print '\nMagField = %f deciTesla \n'%(MagFieldValue)
```

Comment:
why removing this?

Reply:
Because of this change:
https://github.com/cms-sw/cmssw/pull/31009/commits/c8d92328d14646f48a46de527fc741df5ffa743a#diff-19bf9e8388e260aba3f31dde7486846b

The other way I can go is to have the print statement and include MagField=0.0 in the unit test. Would you prefer that? I didn't do it in default because if one uses the Fullname method that passing the MagField has no advantage. Not passing it on the other hand leads to misleading messages in the unit test.

Reply:
The local variabile 'MagFieldValue' is then useless. Why keeping it in the first place? 
The way the B-field is passed to load the conditions and to create the output file name is not very transparent. I think it would be useful to have the bfield value passed in the python configuration actually cross-checked against the value stored in the input files (i.e. in the C++)

Reply:
Ok, I've reverted to the earlier stage. There is no good way to cross-check against the real value of the B-field used in the input  text files, it's just that the cluster shapes look different. There is one info in the header, but that's also just for bookkeeping purposes. So I think this cross-check is not doing what we really want to do.

Reply:
I think cross-checking the header against the value passed in the python configuration is acceptable.

Reply:
OK, I've done that.